project/src/pages/property-page/property-page.tsx

```diff
@@ -145,18 +156,19 @@ export default function PropertyPage({offers, reviews}: PropertyPageProps): JSX.
                   </span>
                 </h2>
                 <ul className="reviews__list">
-                  {reviews.map((item) => <ReviewCard key={item.id} review={item} />)}
+                  {sortedReviews.slice(0, 10)
+                    .map((item) => <ReviewCard key={item.id} review={item} />)}
                 </ul>
                 <ReviewForm />
               </section>
             </div>
           </div>
-          < Map mapClassName={mapClassName.Property} city={offers[0].city} points={offers} />
+          <Map mapClassName={mapClassName.Property} city={offers[0].city} points={nearOffers} selectedPoint={offer} />
         </section>
         <div className="container">
           <section className="near-places places">
             <h2 className="near-places__title">Other places in the neighbourhood</h2>
-            <CardsList offers = {offers} cardClassName={CardClassName.NearPlaces}/>
+            <CardsList offers = {nearOffers} cardClassName={CardClassName.NearPlaces} onListItemHover={() => undefined}/>
```

Comment:
Коллбек не следует передавать совсем
```suggestion
            <CardsList offers={nearOffers} cardClassName={CardClassName.NearPlaces} />
```